### ISSUE TITLE
Update the data structure and what is signed by witness proofs

### ISSUE THREAD

**Issue by swcurran** [decentralized-identity/didwebvh#152]
The data model for the `did-witness.json` file is not correct in the spec. right now. Will correct it.

In addition it was agreed that we would:

- make the `versionId` the object that gets signed -- not the entry.
- The embedded `proof` is an array of proofs from witnesses signing the `versionId`
  - This matches the Data Integrity model of the proof being added to a data structure it is signing.
  - This also allows the entire set of witness proofs to be verified using only the data in the `did-witness.json` file.
  - This would mean that there is no way to know if the witness actually verified a new DID Entry before approving it, or did they just blindly signed the `versionId` value. **Is that OK?**
- When deleting proofs, if a `versionId` has no proofs associated with it, it can be deleted from the file.

Please confirm that this is right @andrewwhitehead @PatStLouis @brianorwhatever and others.  @brianorwhatever (or someone) -- please let me know the data model for the `did-witness.json` file.

**Comment by brianorwhatever** [comment]
Yeah, that looks correct to me. I think this is the data model we agreed on

```
[
  {
    "versionId": "1-Qmba111111...",
    "proof": [{ ... }, { ... }]
  },
  {
    "versionId": "2-Qzmb222222...",
    "proof": [{ ... }, { ... }]
  }
]
```
